.github/workflows/altair-app-deploy-dev.yml

```
@@ -8,23 +8,23 @@ on:
       - 'fabric/**' # dependency
       - 'centrifuge-js/**' # dependency
       - '.github/workflows/nft-studio-build-and-deploy.yml' # dependency
-      - '.github/workflows/nft-studio-deploy-to-dev.yml'
+      - '.github/workflows/altair-app-deploy-dev.yml'
 
 jobs:
   dev:
     uses: centrifuge/apps/.github/workflows/nft-studio-build-and-deploy.yml@main
     with:
       netlify-alias: ''
       ipfs-gateway: https://altair.mypinata.cloud/
-      collator-wss-url: wss://fullnode-collator-2.development.cntrfg.com
-      relay-wss-url: wss://fullnode-relay-2.development.cntrfg.com
+      collator-wss-url: wss://fullnode.development.cntrfg.com
+      relay-wss-url: wss://fullnode-relay.development.cntrfg.com
       subscan-url: https://rococo.subscan.io
       whitelisted-accounts: ''
       network: altair
-      subquery-url: https://api.subquery.network/sq/centrifuge/pools
+      subquery-url: https://api.subquery.network/sq/centrifuge/pools-demo
```

I think should also be
```suggestion
      subquery-url: https://api.subquery.network/sq/centrifuge/pools
```